docs/data/material/components/autocomplete/autocomplete.md

```diff
@@ -227,6 +227,11 @@ The `renderInput` prop allows you to customize the rendered input.
 The first argument of this render prop contains props that you need to forward.
 Pay specific attention to the `ref` and `inputProps` keys.
 
+:::warning
+If you're using a custom input component inside the Autocomplete, you might need to include `forwardRef`. 
+When incorporating the mui's `TextField` component, you should pass `ref` to `inputRef`.
```

Comment:
"mui" is not a great descriptor in this case. Instead, let's add a link to the Text Field doc so there's no confusion about what we're referring to.

```suggestion
When incorporating the [Text Field](/material-ui/react-text-field/) component, you should pass `ref` to `inputRef`.
```

Reply:
That feels better, thank you 😄